app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -62,6 +62,11 @@ angular.module('Bastion.components').factory('Nutupane',
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
 
+            //Additional param to override initial load
+            self.overrideAutoLoad = params.overrideAutoLoad || false;
```

Comment:
I would prefer to see this set on nutupane itself, similar to the (now defunct) `masterOnly` property.